modules/organization-policy/README.md

```diff
@@ -0,0 +1,166 @@
+# Google Cloud Organization Policy
+
+This module allows creation and management of [GCP Organization Policies](https://cloud.google.com/resource-manager/docs/organization-policy/org-policy-constraints) by defining them in a well formatted `yaml` files or with HCL.
+
+Yaml based factory can simplify centralized management of Org Policies for a DevSecOps team by providing a simple way to define/structure policies and exclusions.
+
+## Example
+
+### Terraform code
+
+```hcl
+# using configuration provided in a set of yaml files
+module "org-policy-factory" {
+  source = "./modules/organization-policy"
+
+  config_directory = "./policies"
+}
+
+# using configuration provided in the module variable
+module "org-policy" {
+  source = "./modules/organization-policy"
+  
+  organization_policies = {
+    "folders/1234567890" = {
+        "constraints/iam.disableServiceAccountKeyUpload" = {
+          rules = [
+            {
+                enforce = true
+            }
+          ]
+        }
+    },
+    "organizations/1234567890" = {
+      "run.allowedIngress" = {
+        rules = [
+          {
+            condition = {
```

Comment:
This is a bit complex to use, at this point what's the benefit of using a module, versus using individual resources? Can we find a way of making the interface simple?

Reply:
I will look into some options, initially I thought about simply following the API spec which makes it simpler for a developer to match and fill.

Reply:
I would probably avoid `values` block and remove `allow_all`/`deny_all`
```
rules = [
  {
  allow = [] # stands for all
  },
  {
    deny = ["some_value"] # only some values
  }
]
```
Don't see a way to simplify `condition` block tho